doc/source/configuration.rst

```diff
@@ -74,6 +74,15 @@ classes that load externally stored data.
             module: 'databroker.assets.path_only_handlers'
             class: 'RawHandler'
 
+Optionally, you may set a root_map, which comes in handy when the handlers
+involves mounted files that have been moved to a different mount point in the
+file system.
+.. code-block:: yaml
+    root_map : ['old_root', 'new_root']
```

Comment:
I think we need to be able to support many re-mappings, not just one.  Why not let rootmap be a dict?